packages/cozy-harvest-lib/src/locales/en.json

```diff
@@ -1,5 +1,4 @@
 {
-  "loading": "Loading",
```

Comment:
should we add `'modal.addAccount.button`?

Reply:
and `modal.konnector.create_account` is still used?

Reply:
`modal.addAccount.button` already exists, and `modal.konnector.create_account` was already removed ;)